library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -252,7 +252,8 @@ impl Command {
     #[cfg(not(any(
         target_os = "macos",
         target_os = "freebsd",
-        all(target_os = "linux", target_env = "gnu")
+        all(target_os = "linux", target_env = "gnu"),
+        all(target_os = "linux", target_env = "musl"),
```

Comment:
I think these can drop the `target_env`.

Reply:
There is `uClibc` env which probably does not suport this.

Reply:
The uClibc is the reason why I left target_env. There are no bindings in libc crate, additionally the implementation doesn't satisfy our requirements, i.e., posix_spawn should fail on exec failure.

Reply:
OK, fair enough.